clarify what "preview" means. (#10376)
flutter/website: src/content/perf/impeller.md

### Android

> Impeller is available behind a flag on Android
> devices that support Vulkan.

Reviewer: **chore:** Provide the explanation of "preview" that the original issue requested.

```suggestion
On Android devices that support Vulkan,
you can test and experiment with Impeller.

Impeller might not be suitable for production
environments at this time. 
```

:::secondary Does your device support Vulkan?
You can determine whether your Android device
supports Vulkan at [checking for Vulkan support][vulkan].
:::

> You can preview Impeller on Vulkan-capable Android devices
> by passing `--enable-impeller` to `flutter run`:

Reviewer: **suggestion:** 
Condition, then instruction per [GSG](https://developers.google.com/style/sentence-structure). 
Write in active voice per [GSG](https://developers.google.com/style/voice).

```suggestion
To preview Impeller on Vulkan-capable Android devices,
pass the `--enable-impeller` option to `flutter run`.
```
